app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -412,6 +437,21 @@ class RawSubmissionTable extends React.Component {
     );
   };
 
+  toggleShowInactiveGroups = event => {
+    // depends if the function is called organically or not
```

Comment:
This isn't good. If you are having trouble triggering this with testing, you should first switch to using react-testing-library over enzyme

Reply:
This was actually because this functionality needs to be triggered in the `fetchData` function above and I didn't want to duplicate the code, but `event` is not defined when you call it manually so I passed `null`; hence the need for a check like this. Do you have any suggestions on how I can do this cleanly without having to resort to this type of check?

Reply:
@pranavrao145 ah, great. In this case make it more explicit by modifying the function to take in a boolean, and then make sure to pass in the appropriate argument where it's being used.